about/jobs/template.md

```diff
@@ -0,0 +1,81 @@
+# Template for your LinkedIn
+
+We want to be promote ourselves and FAC as _much_ as we can on our LinkedIn profiles and encourage people to contact us whether it be to source candidates for vacancies or freelance work.
+
+This way, we'll _all_ have _many_ choices for what to do when we're in a position to go into a full-time job or work on freelance projects.
+
+First step is sorting out our profiles. If you haven't got a LinkedIn, [get one](https://www.linkedin.com/start/join)!   
+
+Then get a profile picture where you can see your face! See Josh below:
+
+![Josh](https://files.gitter.im/hdrdavies/zUA6/rsz_josh.png)
+
+This makes you **so** much more approachable. Nelson took a load of piccies of us which you can use if you can't source a suitable headshot. See tips [here](https://business.linkedin.com/talent-solutions/blog/2014/12/5-tips-for-picking-the-right-linkedin-profile-picture)
+
+Profile template to follow:
+
+https://www.linkedin.com/in/hdrdavies
+
+1. Call yourself a 'Full Stack Javascript Developer' in the title. (Recruiters really don't spend a lot of time looking at a profile and often *just* look at the title). A full stack developer sounds a lot more appealing to a recruiter than a 'Trainee', or a 'Web Developer in Training'  (unfortunately)
+
+2. Make sure your area of work is appropriate: 'Information Technology and Services' like Josh above, or 'Internet' etc.
+
+3. Complete any other relevant information in your profile, such as **location**, **education** and **work experience** (emphasise *anything* which is tech based, and consider saying *less* about anything which isn't). You can also say how great you are with prizes you've won and languages you speak, but this doesn't get looked at as much as the top of your profile. Add skills for your buddies to endorse, but these *often* get overlooked too
+
+4. In your summary, put the following:
```

Comment:
I would phrase this as an _example_ of what you can put in rather than 'this is what you should do' :wink: